spec/requests/api/v1/short_form_spec.rb

```diff
@@ -38,10 +40,71 @@
 
   # ---- end Jasmine fixtures
 
+  describe 'show_application' do
+    before do
+      Api::V1::ShortFormController.any_instance
+                                  .stub(:user_can_access?).and_return(true)
+      Api::V1::ShortFormController.any_instance
+                                  .stub(:map_listing_to_application)
+                                  .and_return(true)
+    end
+
+    it 'returns an application object' do
+      VCR.use_cassette('shortform/show_application') do
+        get '/api/v1/short-form/application/a0tf0000000xw9pAAA.json', {}, @auth_headers
+      end
+      json = JSON.parse(response.body)
+      expect(response).to be_success
+      expect(json).not_to be_nil
+    end
+  end
+
+  describe 'show_listing_application_for_user' do
+    before do
+      Api::V1::ShortFormController.any_instance
+                                  .stub(:find_listing_application).and_return(true)
+      Api::V1::ShortFormController.any_instance
+                                  .stub(:find_application_files)
+                                  .and_return(true)
+    end
+
+    it 'returns successful response' do
+      url = '/api/v1/short-form/listing-application/a0Wf0000003j03WEAQ.json'
+      VCR.use_cassette('shortform/show_application') do
+        get url, {}, @auth_headers
+      end
+      expect(response).to be_success
+    end
+  end
+
+  # describe 'submit_application' do
+  # RSPEC IS CONVERTING THE BOOLEAN TO STRINGS, SALESFORCE REJECTING PARAMS
```

Comment:
hm it does make sense that the params are converted to strings, anything you post via a request will be strings. I can take a look at it and see what you mean.